test/cypress/integration/user/dashboard_spec.js

```diff
@@ -1,25 +1,42 @@
 describe('Dashboard', () => {
-  const selfServiceUsers = require('../../../fixtures/config/self_service_user.json')
+  const userExternalId = '7d19aff33f8948deb97ed16b2912dcd3'
+  const gatewayAccountId = '666'
+  const serviceName = 'Test Service'
 
   beforeEach(() => {
-    cy.setCookie('session', Cypress.env('encryptedSessionCookie'))
-    cy.setCookie('gateway_account', Cypress.env('encryptedGatewayAccountCookie'))
+    cy.task('getCookies', {
+      user_external_id: userExternalId,
+      gateway_account_id: gatewayAccountId
+    }).then(cookies => {
+      cy.setCookie('session', cookies.encryptedSessionCookie)
+      cy.setCookie('gateway_account', cookies.encryptedGatewayAccountCookie)
+    })
+
+    cy.task('setupStubs', [
+      {
+        name: 'getUserSuccess',
+        opts: {
+          gateway_account_ids: [gatewayAccountId.toString()],
+          service_name: serviceName
+        }
+      },
+      {
+        name: 'getGatewayAccountSuccess',
+        opts: { gateway_account_id: gatewayAccountId }
+      }
+    ])
```

Comment:
This sets up the stubs that are loaded into Mountebank and used throughout the test. The stubs are cleared after each test.

The stubs are defined in cypress/plugins/stubs.js